Tests/SourceKitLSPTests/SwiftSignatureHelpTests.swift

```diff
@@ -650,8 +657,14 @@ final class SwiftSignatureHelpTests: XCTestCase {
     XCTAssertEqual(
       signature.parameters,
       [
-        ParameterInformation(label: .offsets(start: 8, end: 21)),
-        ParameterInformation(label: .offsets(start: 23, end: 34)),
+        ParameterInformation(
+          label: .offsets(start: 8, end: 21),
+          documentation: .markupContent(MarkupContent(kind: .markdown, value: "The first value"))
+        ),
+        ParameterInformation(
+          label: .offsets(start: 23, end: 34),
+          documentation: .markupContent(MarkupContent(kind: .markdown, value: "The second value"))
+        ),
```

Comment:
Maybe we have this already somewhere but if not, could you add a test case that uses external + internal parameter names, eg.

```swift
/// - number: The number to add 1 to
func addOne(to number: Int) -> Int { number + 1 }
```